spec/services/rate_limit_checker_spec.rb

```diff
@@ -20,14 +43,6 @@
         expect(rate_limit_checker.limit_by_action("comment_creation")).to be(true)
       end
 
-      it "sends a slack message if a user leaves too any messages" do
```

Comment:
We no longer send messages to slack for rate limit hits, this was a false positive spec